internal/core/src/index/ScalarIndexSort-inl.h

```diff
@@ -101,10 +101,10 @@ ScalarIndexSort<T>::Load(const BinarySet& index_binary, const Config& config) {
 }
 
 template <typename T>
-inline const TargetBitmapPtr
+inline const TargetBitmap
 ScalarIndexSort<T>::In(const size_t n, const T* values) {
     AssertInfo(is_built_, "index has not been built");
-    TargetBitmapPtr bitset = std::make_unique<TargetBitmap>(data_.size());
+    TargetBitmap bitset(data_.size());
```

Comment:
why change this? is any benefits can get?

Reply:
YES, this bitmap as a result array, but it's [] operator is not simd friendly because this operator must find the bit in a block